.github/workflows/debian_11.yml

```diff
@@ -0,0 +1,77 @@
+name: debian_11
+
+on: [push, pull_request]
+
+env:
+  CI_MAKE: make -f .gitlab.mk
+
+jobs:
+  # By default jobs on push and pull_request filters run duplicating each other.
+  # To avoid of it used additional external module 'skip-duplicate-actions'.
+  # Used option 'concurrent_skipping' to skip duplicating jobs. Check info:
+  # https://github.com/marketplace/actions/skip-duplicate-actions#concurrent_skipping
+  pre_job:
+    continue-on-error: true
+    runs-on: ubuntu-latest
+    # Map a step output to a job output
+    outputs:
+      should_skip: ${{ steps.skip_check.outputs.should_skip }}
+    steps:
+      - id: skip_check
+        uses: fkirc/skip-duplicate-actions@master
```

Comment:
(If we'll use an external action, it is the rule of thumb to stick with certain tag, like `@v3.3.0`. This way we'll not get surprised by some behaviour change.)

Reply:
Right, better to use some version instead of changing trunk. Any way we'll avoid of use of the module due to PR #5666

Reply:
PR #5666 committed.